portal/templates/redesign/base_new.html

```diff
@@ -256,5 +256,13 @@
 </script>
 {% endblock scripts %}
 
+<script type="text/javascript">
+    $('.logout').click(function(){
+        $(this).toggleClass("button--secondary");
+        $(this).toggleClass("button--secondary--dark");
```

Comment:
what is secondary--dark? I don't think @ruthdw would approve this naming pattern :p - please write an issue :)